### PR TITLE
fixed rules for naming variables

### PR DESCRIPTION
**Pull request by venthur**
Rules for naming variables:

    PoolManager and ProxyManager should be http
    ConnectionPool should be pool
    Connection should be conn
    HTTPResponse should be resp (ie resp = http.request("GET", "https://example.com"))
    Only use example.com or httpbin.org for example URLs

There was only a few occurrences to fix.

related: #2223

### CODE REVIEW

File: docs/advanced-usage.rst [hunk 1]
```diff
@@ -117,8 +117,8 @@ You can use :class:`~poolmanager.ProxyManager` to tunnel requests through an
 HTTP proxy::
 
     >>> import urllib3
-    >>> proxy = urllib3.ProxyManager('http://localhost:3128/')
-    >>> proxy.request('GET', 'http://google.com/')
+    >>> http = urllib3.ProxyManager('http://localhost:3128/')
+    >>> http.request('GET', 'http://google.com/')
```
Comment (pquentin) [inline docs/advanced-usage.rst:121]: @sethmlarson I think I liked the `proxy` name which makes it clear that the request is going through a proxy, but don't have any strong opinion. What's the rationale for using `http` here?

**Review by pquentin — APPROVED** [review]

File: docs/advanced-usage.rst (hunk 1 above)
Comment (sethmlarson) [reply docs/advanced-usage.rst:121]: I'm ++ on using `proxy` here, I'll update the rules.

**Review by sethmlarson — APPROVED** [review]
LGTM

**Comment by sethmlarson** [comment]
Btw @venthur:
- If you're interested we have a public Discord where we're discussing this effort, if possible it'd be nice to coordinate these changes.
- Instead of writing "partially fixes #XYZ" in the issue write "related to #XYZ", etc. The words "fixes #XYZ" makes the issue get closed after I merge the PR.

**Pull request merged**